src/omnisharp/EventType.ts

```diff
@@ -82,7 +82,8 @@ export enum EventType {
     ProjectDiagnosticStatus = 75,
     DotNetTestRunInContextStart = 76,
     DotNetTestDebugInContextStart = 77,
-    TelemetryErrorEvent = 78
+    TelemetryErrorEvent = 78,
+    OmnisharpServerRequestCancelled = 79
```

Comment:
Nit: consider leaving a trailing comma here, so that future PRs that add elements don't have to modify this line.